core/test/results/sample_v2.json

```diff
@@ -1027,6 +1027,18 @@
         ]
       }
     },
+    "third-party-cookies": {
+      "id": "third-party-cookies",
+      "title": "Avoids third-party cookies",
+      "description": "Support for third-party cookies will be removed in a future version of Chrome. [Learn more about phasing out third-party cookies](https://developer.chrome.com/en/docs/privacy-sandbox/third-party-cookie-phase-out/).",
+      "score": 1,
+      "scoreDisplayMode": "binary",
+      "details": {
+        "type": "table",
+        "headings": [],
+        "items": []
+      }
+    },
```

Comment:
Done, bit of noise in the sample changes but not too bad

Reply:
Update: I reverted this change because we will need to regen the DT log as well to prevent some subtle errors. I think the smoke test is sufficient until we do full artifacts regen.